CHANGES.md

```diff
@@ -77,6 +77,11 @@ OpenSSL 3.0
 
    *Tomáš Mráz*
 
+ * Removed RSA padding mode for SSLv23 (which was only used for
+   SSLv2)
```

Comment:
Maybe also mention that the `rsautl -ssl` is removed and those two public functions from the RSA API are removed as well.